### PR TITLE
Overhaul Terminal Pro docs - WIP

### PR DESCRIPTION
**Pull request by DidierRLopes**

### CODE REVIEW

**Comment by minhhoang1023** [comment]
## 1. Terminology
We should be more consistent with the wording, if possible. We no longer use the word "Native Integration" anywhere on the product, so i don't think we should mention it on the Documentation page. 

Screenshot 2024-09-02 at 11 02 28

File: content/pro/openbb-copilot.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+For example, if you ask about the price-to-earnings (P/E) ratio, and then follow
+up by asking about other important financial ratios, OpenBB Copilot will
+remember the context and suggest relevant ratios:
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/pe_ratio_chat_example.jpg" alt="chat history openbb copilot" width="40%" height="40%" />
+
+If you'd like to clear the history of the current conversation, you can do so by
+clicking on the trashcan icon in the OpenBB Copilot chatbox in the upper right
+corner. It is usually a good idea to clear the chat history when you have a new
+question that is unrelated to your current conversation.
```
Comment (mnicstruwig) [inline content/pro/openbb-copilot.md:55]: @IgorWounds we'll need to add a section on new chats / deleting chats, etc. and how they work.

File: content/pro/openbb-copilot.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+## General question answering
```
Comment (mnicstruwig) [inline content/pro/openbb-copilot.md:57]: @IgorWounds I think we should also add an image to this section.

File: content/pro/openbb-copilot.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+Copilot will retrieve the Company News widget data on your dashboard (provided
+the widget has been added to the dashboard) to provide an answer.
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/aapl_+news_chat_example.jpg" alt="aapl news openbb copilot" width="80%" height="80%" />
+
+When querying OpenBB Copilot, it is important to understand which data OpenBB
+Copilot used to answer your query. As a result, OpenBB Copilot will always cite the data
+source it referenced when responding to your query. 
+
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/citation_chat_example.jpg" alt="citations openbb copilot" width="80%" height="80%" />
```
Comment (mnicstruwig) [inline content/pro/openbb-copilot.md:86]: @IgorWounds Let's add an entire section dedicated to citations and status updates, and instead just focus on retrieving data from the dashboard in this section.

File: content/pro/openbb-copilot.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+
+| ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation_button.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_creation.jpg) | ![widget generate button openbb copilot](https://openbb-assets.s3.amazonaws.com/docs/copilot/table_widget_created.jpg) |
+|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------:|
+
+## Querying specific widgets only
```
Comment (mnicstruwig) [inline content/pro/openbb-copilot.md:105]: @IgorWounds This section should come before the "Creating widgets" section. Let's group everything together in logical order.

I'm expecting something like:

- Introduction
- How chats work
- Retrieving data
  - From the dashboard
  - Explicitly as context
  - User files
    - (with subsections for directly dragging-and-dropping vs. adding as a widget)
  - Direct retrieval
    - (with subsection for custom endpoints)
  - Searching the Web 
- Citations, Artifacts, and Status Updates
- Creating widgets from OpenBB Copilot outputs
- Prompting Guidelines (<-- link to OpenAI's prompting tips page, or summarize it here)

**Review by andrewkenreich — APPROVED** [review]

**Pull request merged**